src/Microsoft.AspNet.Mvc.ModelBinding/Internal/ModelMetadataAttributeHelper.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding.Internal
+{
+    internal class ModelMetadataAttributeHelper
```

Comment:
this class has no state and is used in a way that prevents replacement.  either define an interface and add an instance (or type) to the MVC configuration (perhaps overkill) or make this a `static` class.  this point is moot if this class goes away and we have both model metadata and validation use `ModelAttributes`, which is already `static`